ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -1,1136 +1,1417 @@
 {
```

Comment:
<p>Since the whole notebook is a diff (I bet the indentation changed... I just did that to someone myself :) I'm not sure I caught all the changes, but I saw the service account related ones as well as the new cleanup section. That all looks good. I should be able to look through the others tomorrow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
thanks